app/workers/edition_batch_link_checker.rb

```diff
@@ -0,0 +1,31 @@
+# Calls the Link Checker API to verify all links in an edition per organisation
+require "sidekiq-scheduler"
```

Comment:
Do we need this in the worker class directly? Can we move it into the initialisation for Sidekiq?

Reply:
This is the suggested usage from the [repo](https://github.com/moove-it/sidekiq-scheduler#hello-world) follows how we've used in the [Link Checker API](https://github.com/alphagov/link-checker-api/blob/master/app/workers/cleanup_worker.rb#L1)

Reply:
😢 That's annoying, it looks strange having to put it in the worker when the worker doesn't care if it's scheduled or not!